README.md

```diff
@@ -93,6 +67,65 @@ Usage
 @include report;
```

Comment:
@ericam One thing I'm not clear on is what this line does exactly. It doesn't seem to be needed (or to do anything) when using True with node-sass and Mocha. I wasn't sure what to say about whether to include it or not.